package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@folio/plugin-eusage-reports",
-  "version": "2.3.0",
```

Comment:
We don't need to make a new major release of the plugin because we're _adding_ support for a new version of Stripes (see below): that would only be necessary if we were _removing_ support for an old version, which we don't need to do.